projects/ontimize-web-ngx/src/lib/components/table/o-table.component.ts

```diff
@@ -1555,21 +1558,22 @@ export class OTableComponent extends OServiceComponent implements OnInit, OnDest
     this.queryData(void 0, queryArgs);
   }
 
-  handleClick(item: any, rowIndex: number, $event: MouseEvent) {
+  handleClick(item: any, column: string, rowIndex: number, $event: MouseEvent) {
     this.clickTimer = setTimeout(() => {
       if (!this.clickPrevent) {
-        this.doHandleClick(item, rowIndex, $event);
+        this.doHandleClick(item, column, rowIndex, $event);
       }
       this.clickPrevent = false;
     }, this.clickDelay);
   }
```

Comment:
This method is been removed from the template, Is it still necessary here?
If not needed, maybe it is a good idea to rename `handleCellClick` to `handleClick` too.